### PR TITLE
fix: Change title in EventDetailsFragment to Event name

### LINKED ISSUE #167: Title bug in EventDetailsFragment
This occurs after EventDetailsFragment opens on click of event in Search section
screenshot_20180620-032008

Working on it :+1:

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #167

videotogif_2018 06 23_13 09 51

### CODE REVIEW

**Comment by dreadpool2** [comment]
@iamareebjamal What are your views?

**Comment by dreadpool2** [comment]
@iamareebjamal What should be done?

**Comment by iamareebjamal** [comment]
Let it be like EventBrite

File: app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt [hunk 1]
```diff
@@ -59,6 +61,17 @@ class EventDetailsFragment : Fragment() {
...
         eventViewModel.loadEvent(eventId)
 
+        //Set toolbar title to event name
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            rootView.nestedContentEventScroll.setOnScrollChangeListener { _, _, scrollY, _, _ ->
+                if (scrollY > rootView.eventName.height + rootView.logo.height) {
+                    activity?.supportActionBar?.title = eventShare.name
+                } else {
+                    activity?.supportActionBar?.title = " "
+                }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: What does this do?

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: When we scroll downwards, after we have scrolled a the length of event name and image, event name will be shown. If we scroll upwards, event name disappears

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: Write this in comment

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: Added comment :+1:

**Pull request merged**